app/views/articles/_video_player.html.erb

```diff
@@ -1,16 +1,30 @@
-<div itemscope itemtype="http://schema.org/VideoObject" class="video-player-header">
+<div class="video-player-header">
   <% if meta_tags %>
-    <meta itemprop="uploadDate" content="<%= article.published_at %>" />
-    <meta itemprop="name" content="<%= article.title %>" />
-    <meta itemprop="description" content="<%= article.description %>" />
-    <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
-    <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
+    <meta content="<%= article.published_at %>" />
+    <meta content="<%= article.title %>" />
+    <meta content="<%= article.description %>" />
+    <meta content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
+    <meta content="<%= article.video_source_url %>" />
     <% minutes, seconds = article.video_duration_in_minutes.split(":") %>
-    <meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
+    <meta content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
     <div id="video-player-source" data-source="<%= article.video_source_url %>"></div>
   <% end %>
   <script src="//content.jwplatform.com/libraries/b1zWy2iv.js" async></script>
   <div id="video-player-<%= article.id %>" class="video-player"></div>
+
+  <script type="application/ld+json">
```

Comment:
I tested the result of this with https://search.google.com/structured-data/testing-tool/u/0/  and it flagged `A value for the uploadDate field is required.`

Test JSON:

```json
{
  "@context": "https://schema.org",
  "@type": "VideoObject",
  "url": "http://example.com",
  "datePublished": "2020-04-01T12:16:23Z",
  "name": " Nine Coaches Waiting Qui explicabo",
  "description": "Umami selvage hammock. Master fashion axe heirloom.           A   Heirloom try-hard shabby chic. Salv...",
  "thumbnailUrl": "https://dummyimage.com/880",
  "contentUrl": "http://example.com",
  "duration": "PT100M1S"
}
```

so we might want to add that as well.

Reply:
OH! I replaced the `uploadDate` field with `datePublished` on Friday after digging through some docs, but will change this back. 😄